src/components/FwbBlockquote/composables/useBlockquoteClasses.ts

```diff
@@ -0,0 +1,40 @@
+import type { BlockquoteSize, BlockquoteAlign, BlockquoteType } from "../types";
+import { computed, ref, useAttrs } from "vue";
+import { twMerge } from "tailwind-merge";
+
+const defaultBlockquoteClasses = "font-semibold italic text-gray-900 dark:text-white";
+const blockquoteAlignClasses: Record<BlockquoteAlign, string> = {
+  left: "text-left",
+  center: "text-center",
+  right: "text-right",
+};
+const blockquoteSizeClasses: Record<BlockquoteSize, string> = {
+  sm: "text-lg",
+  md: "text-xl",
+  lg: "text-2xl",
+};
```

Comment:
as stated above... this should be achieved by tailwind classes directly on the component